package.json

```diff
@@ -145,6 +158,7 @@
     "moduleNameMapper": {
       "^react-native$": "react-native-web",
       "^.+\\.module\\.(css|sass|scss)$": "identity-obj-proxy",
+      "(lib-vendor/.*)": "victory-vendor/$1",
```

Comment:
Why do we actually need this line?

Reply:
Thus I needed to add because of an error in one of the d3 packages. Comment it out and try to npm i and you will see what fails.